components/teams-service/server/v2/server.go

```diff
@@ -244,6 +244,17 @@ func (s *Server) UpgradeToV2(ctx context.Context,
 	return &teams.UpgradeToV2Resp{}, nil
 }
 
+// ResetToV1 removes all projects from teams.
```

Comment:
For consistency with last file...
```suggestion
// ResetToV1 removes all projects from every team.
```